### PR TITLE
Update Stripe Link inbox notification wording

### PR DESCRIPTION
**Pull request by mordeth**
#### Changes proposed in this Pull Request
Update Stripe Link inbox notification wording to not exceed the maximum allowed characters.

Screenshot 2022-11-01 at 9 36 29

#### Testing instructions
- Navigate to 'Plugins' -> Deactivate and then activate the WooCommerce Payments plugin
- Navigate to `WooCommerce` -> `Home`.
- Verify the Stripe Link inbox notification text is not cut out.

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

**Post merge**

- [ ] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) following [these instructions](https://github.com/Automattic/woocommerce-payments/wiki/How-to-write-good-manual-testing-scenarios) : _Add link here / 'QA Testing Not Applicable'_
- [ ] Add or update [critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Critical-flows) and [testing instructions for critical flows](https://github.com/Automattic/woocommerce-payments/wiki/Testing-instructions-for-critical-flows), if applicable.
- [ ] Add what's changed (description, screenshot, demo videos etc.) to the release announcement post, if applicable.

### CODE REVIEW

**Review by dpaun1985 — APPROVED** [review]
LGTM!

**Review by nagpai — CHANGES_REQUESTED** [review]
LGTM! One small grammar change suggested by language tool  - Add a comma before `so` in `... shipping details, so they can check out...`

## Screenshot from language tools:
image

(sorry I forgot to add the comment using the +  in the review section).

**Review by dmvrtx — APPROVED** [review]
Looks good to me. Maybe we can add a link to some page with Link marketing later on.

File: includes/notes/class-wc-payments-notes-set-up-stripelink.php [hunk 1]
```diff
@@ -75,7 +75,7 @@ public static function get_note() {
 		$note       = new $note_class();
 
 		$note->set_title( __( 'Increase conversion at checkout', 'woocommerce-payments' ) );
-		$note->set_content( __( 'Reduce cart abandonment and create a frictionless checkout experience with Link by Stripe. Link autofills your customer’s payment and shipping details so they can check out in just six seconds with the Link optimized experience. That’s 9x faster than shoppers who don’t use Link. Link increases conversion rates by over 7% for logged-in Link customers.', 'woocommerce-payments' ) );
```
Comment (anu-rock) [inline includes/notes/class-wc-payments-notes-set-up-stripelink.php:78]: > That’s 9x faster than shoppers who don’t use Link...

I think the hard numbers will increase the appeal of the overall message and make it more convincing.

Here's the original message rephrased in 250 chars:

`Reduce cart abandonment: create a frictionless checkout experience with Link by Stripe. Link autofills your customer’s payment and shipping details so they can check out 9x faster than non-Link shoppers. Link customers are 7% more likely to convert.`

Comment (anu-rock) [reply includes/notes/class-wc-payments-notes-set-up-stripelink.php:78]: The message's effectiveness can further be increased by providing a link for more details and instructions on how to enable it:

https://woocommerce.com/document/payments/link-by-stripe/

**Review by anu-rock — CHANGES_REQUESTED** [review]
I'm not sure how much control we as developers have over the messaging, but I've left a couple of comments that may improve its effectiveness.

File: includes/notes/class-wc-payments-notes-set-up-stripelink.php (hunk 1 above)
Comment (dpaun1985) [reply includes/notes/class-wc-payments-notes-set-up-stripelink.php:78]: @anu-rock the message above was agreed on business side. I don't think we should change it. Context: pdjVC1-1aT-p2#comment-1737 .

**Pull request merged**